src/XrdHttp/XrdHttpProtocol.cc

```diff
@@ -2850,6 +2852,17 @@ int XrdHttpProtocol::xauth(XrdOucStream &Config) {
   return 0;
 }
 
+int XrdHttpProtocol::xkeepheadercase(XrdOucStream &Config) {
+  char *val = Config.GetWord();
+  if (!val || !val[0])
+  {eDest.Emsg("Config", "keepheadercase argument not specified"); return 1;}
+
+  if(!strcmp("yes",val)) {
+    keepHeaderCase = true;
+  }
```

Comment:
There should be a else keepHeaderCase = false; to take care of the case
http.keepheadercase yes
http.keepheadercase no
Yes, people do that.

Reply:
That's totally correct, just force-pushed with that change included!

Reply:
Something wennt wrong. The change does not appear in your new ush. In fact, the commit count is still 1. So, it shows the change has not been done. Can you try again?

Reply:
Sorry yes, pushed the wrong branch ;) Now it should be good...